Sources/TripKit/model/TKBookingTypes.swift

```diff
@@ -36,7 +36,7 @@ public enum TKBooking {
       case lock
       case unlock
       case cancel
-      case returntrip
+      case plannext
```

Comment:
Please change this to Swift style:

```suggestion
      case planNext = "plannext"
```